doc/man1/openssl-s_client.pod.in

```diff
@@ -316,39 +319,63 @@ abort the handshake with a fatal error.
 
 Limit verify output to only errors.
 
+=item B<-CAfile> I<file>
+
+A file in PEM format containing trusted CA certificates.
+
+=item B<-CApath> I<directory>
+
+A directory with files in PEM format containing trusted CA certificates.
+This directory must be in "hash format",
+see L<openssl-verify(1)> for more information.
```

Comment:
A link to c_rehash or other way to create such a directory will be useful

Reply:
It turns out the references to `L<openssl-verify(1)>` no more made sense; replaced them by `L<openssl-rehash(1)>`.